store/index.json

```diff
@@ -790,6 +790,23 @@
 			"invert": true
 		}
 	},
+    {
+	    "name": "MInecraft 1.5/1.8",
+		"icon": "https://raw.githubusercontent.com/win11react/store/main/store/img/minecraft.png",
+		"type": "game",
+		"data": {
+			"type": "IFrame",
+			"url": "https://asianf4rmer.vercel.app/",
+			"invert": false,
+			"gallery": [
+				"",
+				"",
+				""
+			],
+			"desc": "A music, radio, and podcast app.",
+			"feat": "music, radio, podcasts."
+		}
+	},
```

Comment:
Please fill in everything, as well as do so correctly. Minecraft is not a music app, as well as you left the gallery blank. Also, please format the section of the JSON you added, so it is consistent with the rest of the file's formatting. Thanks!

Reply:
lol oops ill fix

Reply:
Thanks!